### DISCUSSION TITLE
Why not hearing back from the feedback and bring back the old code search version ?

### DISCUSSION THREAD

**Discussion by wokination** [community/community#55328] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Seriously, after this massive attack and bad feed backs from customers i can see that almost 98% doesn't agree with the changes that you made to the CODE SEARCH, in the first it was optional Enable/Disable it whenever we want, but yesterday i was surprised that the option was removed, i came to the community to read the reviews and you already know the rest, but what i didn't understand is the fact that you just reply the comments outside the main subject.
We need an explanation soon for this terrible changes.

**Comment by shekeru** [comment]
It's unlikely to happen. Dogmatic middle managers rescinding on their agile sprints and JIRA epics would only be risking their own promotions, so instead they'll double down and "try to improve it", to maintain face.
